.github/workflows/image.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Pushing docker images for every pull request is not a good practice, it’s better to push the images only when the commit is pushed to master branch.

Reply:
+1

Reply:
agree, the workflow is only for test now